### ISSUE TITLE
Kubernetes: notify the users about errors

### ISSUE THREAD

**Issue by aermakov-zalando** [zalando/skipper#901]
Right the users can only find out about problems with an Ingress (e.g. unparseable `skipper-routes`) by looking at Skipper logs. This is cumbersome and we should do better: if there are any errors or warnings with an Ingress Skipper should emit a Kubernetes event on the Ingress object.

**Comment by hjacobs** [comment]
Yes, this sounds like a valuable improvement for the user experience.
